client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -79,6 +80,34 @@ class PostActionCounts extends PureComponent {
 		);
 	}
 
+	renderViewCount() {
+		const { viewCount: count, numberFormat, postId, showViews, siteSlug, translate } = this.props;
+		if ( ! count || count < 1 || ! showViews ) {
+			return null;
+		}
+		return (
+			<li>
+				<a
+					href={ `/stats/post/${ postId }/${ siteSlug }` }
+					onClick={ this.onActionClick( 'stats' ) }
+					title={ translate(
```

Comment:
I like that desktop users can hover over to see details about this, but I don't think using `title` on the anchor element is semantically correct, since this text doesn't convey information about the destination of the link.

Maybe nesting a `<span>` with `title` and `aria-label` attributes would be better? This still leaves out touch screen users though...

I wonder if we could use a more descriptive title that is still terse... "Views this month"?